recipes/bigdft/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "bigdft-suite" %}
+{% set version = "1.9.3_rc.2" %}
+{% set build = 0 %}
+{% set mpi = mpi %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/l_sim/bigdft-suite/-/archive/{{ version }}/bigdft-suite-{{ version }}.tar.gz
+  md5: a756c1673c45e7e7b7be3cdf6aef81b1
+  patches:
+    - patches/output.patch
+    - patches/utils.patch
+
+build:
+  number: {{ build }}
```

```suggestion
build:
  run_exports:
     -  {{ pin_subpackage(name|lower) }}
  number: {{ build }}
```
Shipping header files means that you should pin this package according to ABI compatability guarantees.